src/cpp/session/SessionMain.cpp

```diff
@@ -650,7 +650,8 @@ Error rInit(const rstudio::r::session::RInitInfo& rInitInfo)
    error = rstudio::r::json::getRpcMethods(&rMethods);
    if (error)
       return error;
-   for (const json::JsonRpcMethod& method : rMethods)
+   
+   for (json::JsonRpcMethod method : rMethods)
```

Comment:
Is `const &` incorrect here?

Reply:
It seems to be; at least, `-Wrange-loop-analysis` indicates that doing this avoids copies. I don't quite understand why but this makes the compiler happy.